CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ etcd container with this new field.
 
 ### Removed
 
+- Drop flags "masterHost", "cert-file", "key-file", "ca-file".
```

Comment:
I realize we shouldn't use them anymore. We require operator to be a pod and have a service account.